mappings/net/minecraft/data/client/ItemModelGenerator.mapping

```diff
@@ -23,7 +23,11 @@ CLASS net/minecraft/unmapped/C_cfvrpkog net/minecraft/data/client/ItemModelGener
 	METHOD m_llpeiyoc createArmorJson (Lnet/minecraft/unmapped/C_ncpywfca;Ljava/util/Map;Lnet/minecraft/unmapped/C_ncpywfca;)Lcom/google/gson/JsonObject;
 		ARG 1 id
 		ARG 2 textures
+		ARG 3 overrideArmorMaterialId
```

Comment:
I see this comes from `TrimModelData#overrideArmorMaterials` which is proposed, but I can't find the source of the proposal.

I think `armorMaterialOverrides` would be a more natural word order and sound less like a boolean or a method name.

Reply:
i agree, i've updated the names in the record